### DISCUSSION TITLE
How should the documentation be made

### DISCUSSION THREAD

**Discussion by aryaveersr** [timlrx/tailwind-nextjs-starter-blog#77] [Q&A]
How should the documentation be made ?

* Use a documentation theme like just-the docs ?

* Create a documentation site specifically for this project.

* if you want to be more ironic, how about using the blog template with some changes for the docs 😁

**Comment by timlrx** [comment]
I thought the blog intro post itself  / README.md suffice as the documentation and there isn't a need for another one. So the thing is which areas do you feel can be better explained and let's improve those parts.

**Comment by aryaveersr** [comment]
I feel the customization section deserves its own page, basically a guide on what everything does and how to customize when they first clone.

**Reply by timlrx** [reply]
Make sense to me, happy to add it in as a new post.

**Reply by timlrx** [reply]
If you plan on working on this, kindly use the v1 branch as reference
